### PR TITLE
added fast5 loading function

### PR DESCRIPTION
**Pull request by mostwanted7**
Wrote some comments in my commits about things I am unsure about

### CODE REVIEW

File: src/persistence.jl [hunk 1]
```diff
@@ -1 +1,24 @@
...
+    var_range = read(attributes(channel_info)["range"])
+    digitisation = read(attributes(channel_info)["digitisation"])
+    offset = Int(read(attributes(channel_info)["offset"]))
+    raw = read(raw_signal["Signal"]) #Int16 array
+
+    # Normalization
+    normalized_data = []
+    scaling = var_range / digitisation
+    for i in raw
+        x = convert(Float32, (scaling * (i + offset)))
```
Comment (JokingHero) [inline src/persistence.jl]: It is better to use functional programming paradigms as they can be nicely paralalized:
so something like:

normalized_data = Base.map(i -> convert(Float32, (scaling * (i + offset))), raw)

We can later on replace Base with ThredsX for example and have it paralel.

File: src/persistence.jl [hunk 2]
```diff
@@ -1 +1,24 @@
 # Here put file load/file write functions
+function loadfast5(path)
```
Comment (JokingHero) [inline src/persistence.jl]: name function loadnanoporefast5, add defined types path::String, add some function description with intended input/output, add some tests for this function

File: src/persistence.jl [hunk 3]
```diff
@@ -1 +1,24 @@
 # Here put file load/file write functions
```
Comment (JokingHero) [inline src/persistence.jl]: remove this comment I left

File: src/persistence.jl (hunk 2 above)
Comment (JokingHero) [reply src/persistence.jl]: also we need to add this function to exports, then do the tests

**Comment by mostwanted7** [comment]
Made the proposed changes, not sure what else can be tested about this function other than just checking whether it normalizes properly a random value of choice? (Since giving it a 21 000 element array as input in test would be too much?)

**Review by JokingHero — APPROVED** [review]

**Pull request merged**